### ISSUE TITLE
Ability to received (via API or form) Inbound Peering Requests

### ISSUE THREAD

**Issue by tardoe** [peering-manager/peering-manager#169]
### Proposed Functionality
Either expose an externally facing form or an API endpoint to collect peering requests containing the important information: ASN, peering contact, assurance that PeeringDB is up-to-date, exchange location of interest, bilat/PNI type etc. Store these requests and allow admins to mark them as approved / rejected with templated responses.

### Use Case
A large, highly connected network that gets multiple peering requests per-week in un-structured email formats could utilise such a system to track in-bound requests and link them to further automation. 

### External Dependencies
N/A

**Comment by mngan** [comment]
I see the peering requests handled by a separate system, and uses the existing P-M API's to create the sessions and ASN objects as they are approved. Having this be a separate system could possibly allow people not using P-M to hook into their own peering management system, which would provide larger opportunities for adoption. 

I did have a larger vision for a peering portal, but the more I think about it, I kind of see the peering request portal as just a pluggable component, or back end service for the portal. Things like a looking glass could be integrated into this, but some people may already have an existing one they want to use. Traffic stats is a tricky one because everyone handles that differently, so finding a common mechanism to present the data could be tricky (embded grafana graphs? RRD graphs?). The BGP session queries should go through the peering request management system so it can abstract away the peering management system on the back end.

**Comment by gmazoyer** [comment]
I agree with @mngan. What we need to support such external form it to provide the required API endpoints.
